### PR TITLE
Antipattern in CI/CD samples

### PR DESCRIPTION
**Pull request by afleisc**
Added Cloud Build samples for both public and private Github repositories

### CODE REVIEW

File: bigquery-antipattern-recognition/cicd/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Using BigQuery AntiPattern Tool in CI/CD
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/cicd/README.md:1]: can we mice the cicd folder into a subfolder of the root folder. the current folder is for the java code

File: bigquery-antipattern-recognition/cicd/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Overview
+
+One option to deploy the BigQuery Anti Pattern Recognition tool is to run it as part of a CI/CD process to check SQL files present in a Github repository. The following example assums all SQL files are present in one folder.
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/cicd/README.md:5]: asumes

File: bigquery-antipattern-recognition/cicd/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* Package the tool into a container and push it to Artifact Registry.
+* Create a trigger in Cloud Build which is connected to your Github repository.
+* Modify the sample cloudbuild.yaml present in this folder and add it to your repository at the root level. 
+
+## Package the tool into a container and push it to Artifact Registry
+
+1. Create an Artifact Registry Repository, if necessary
+
+    ``` bash
+    gcloud artifacts repositories create $REPOSITORY \
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/cicd/README.md:17]: can you please add the declaration of these variables:

```
export REPOSITORY='my-repository'
export REGION= ...
...
```

File: bigquery-antipattern-recognition/cicd/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Included files (optional): Changes affecting at least one of these files will invoke a build. You can use glob strings to specify multiple files with wildcard characters. Acceptable wildcard characters include the characters supported by Go Match, **, and alternation.
+
+Example: enter input/** if your SQL files are present in the input/ folder.
+
+(Optional) Enable the option to "Send build logs to Github"
+
+## Modify the cloudbuild.yaml sample and add it to your repo.
+
+** Important: Grant your Cloud Build service account the `Artifact Registry Reader` role before continuing.
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/cicd/README.md:45]: could we please ad the gcloud command for this

**Comment by franklinWhaite** [comment]
LGTM let's just fix the minor nit.

**Comment by afleisc** [comment]
Thanks I addressed the comments

**Pull request merged**